content/integrations/tomcat.md

```diff
@@ -3,16 +3,252 @@ title: Datadog-Tomcat Integration
 integration_title: Tomcat
 git_integration_title: tomcat
 kind: integration
-doclevel: basic
+newhlevel: true
+updated_for_agent: 5.8.5
 ---
+# Overview
 
-For information on Tomcat, please see [here][1].
+Get metrics from Tomcat in real time to:
+
+* Visualize your web server performance
+* Correlate the performance of Tomcat with the rest of your applications
+
+# Installation
+
+Metrics will be captured using a JMX connection. We recommend the use of Oracle's JDK for this integration.
```

Comment:
this should work with openjdk as well shouldn't it? do we know what the caveats are?